### ISSUE TITLE
Provide a `RowDVectorView` for `RowDVector`s similar to `DVectorView` for `DVectors`

### ISSUE THREAD

**Issue by Tastaturtaste** [dimforge/nalgebra#1229]
One strength of nalgebra is the ability to check with the type system that the dimensions of Matrices and Vectors match. Even when one dimension is dynamic, the other dimension can still be statically checked. 

For convenience there are type defs for owning row- and column vectors. In addition there are view types for matrices and column vectors. For symmetry with the owning types I would like to have a `RowDVectorView` as a view of a row vector similar to `DVectorView` for column vectors. Currently it's absence feels like a missing piece.

**Comment by Andlon** [comment]
I agree, this seems inconsistent. Would you be up for submitting a PR?

**Comment by Tastaturtaste** [comment]
> I agree, this seems inconsistent. Would you be up for submitting a PR?

Done: #1232
